packages/components/src/SearchInput.tsx

```diff
@@ -52,7 +79,47 @@ class SearchInput extends PureComponent<SearchInputProps> {
       id,
       onKeyDown,
       'data-testid': dataTestId,
+      queryParams,
     } = this.props;
+
+    let matchCountSection;
+
+    if (queryParams && matchCount > 1) {
+      matchCountSection = (
+        <>
+          <Button
+            kind="ghost"
+            className="search-change-button"
+            type="button"
+            onClick={() => {
+              queryParams.changeQueriedColumnIndex('back');
+            }}
+            icon={vsArrowLeft}
+            tooltip="Next match"
+          />
+          <span className="search-change-text">
+            {queryParams.queriedColumnIndex !== undefined &&
+              `${queryParams.queriedColumnIndex + 1} of `}
+            {matchCount}
+          </span>
+          <Button
+            kind="ghost"
+            className="search-change-button"
+            type="button"
+            onClick={() => {
+              queryParams.changeQueriedColumnIndex('forward');
+            }}
+            icon={vsArrowRight}
+            tooltip="Next match"
+          />
+        </>
+      );
+    } else {
+      matchCountSection = matchCount > 0 && (
+        <span className="match_count">{matchCount}</span>
```

Comment:
Class names shouldn't have an underscore in them like this. HTML attributes and CSS properties should be [kebab case](https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case), not [snake case](https://developer.mozilla.org/en-US/docs/Glossary/Snake_case).

Also not sure why you changed the name - should just keep the name `search-match`, and update the existing CSS definition for it (right now it's defined in `SearchInput.scss` but not used anymore).